nntrainer/utils/node_exporter.cpp

```diff
@@ -50,6 +50,25 @@ void Exporter::saveTflResult(const std::tuple<> &props, const LayerV1 *self) {
   createIfNull(tf_node);
 }
 
+static void saveTflWeights(TfOpNode *tf_node, const RunLayerContext &context,
```

Comment:
Nice, could you remove `var_grad::cloneTransposeVariableOnly` within this PR?

Reply:
I will remove it as part of a new commit.